achyut/src/main/java/com/lftechnology/runner/Executer.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.runner;
+
+import java.util.Scanner;
+
+public class Executer {
+
+    private static Scanner userInput;
+    public static int chosenOption;
+
+    public static void main(String[] args) {
+        Menu menu = new Menu();
+        menu.showmenu();
+    }
+
+    public void askForInputAndRun() {
+        /**
+         * @todo Check if user input is empty in all the subsequent classes
+         */
+        userInput = new Scanner(System.in);
+        Executer.chosenOption = userInput.nextInt();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make the enclosing method "static" or remove this set. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS2696)